src/components/KTabs/KTabs.vue

```diff
@@ -11,21 +11,21 @@
         class="tab-item"
         :class="{ active: activeTab === tab.hash }"
       >
-        <div
+        <component
+          :is="tabComponent(tab).tag"
           :aria-controls="hidePanels ? undefined : `panel-${tab.hash}`"
           :aria-selected="hidePanels ? undefined : (activeTab === tab.hash ? 'true' : 'false')"
           class="tab-link"
-          :class="{ 'has-panels': !hidePanels, disabled: tab.disabled }"
+          :class="{ disabled: tab.disabled }"
           role="tab"
-          :tabindex="getAnchorTabindex(tab)"
-          @click.prevent="!tab.disabled ? handleTabChange(tab.hash) : undefined"
-          @keydown.enter.prevent="!tab.disabled ? handleTabChange(tab.hash) : undefined"
-          @keydown.space.prevent="!tab.disabled ? handleTabChange(tab.hash) : undefined"
```

Comment:
question: why are we removing the keyboard events?

Reply:
It was a `div` and now it's becoming either a `button` or an `a` - both elements support keyboard events natively